galleries/users_explain/axes/axes_units.py

```diff
@@ -0,0 +1,282 @@
+# "1980" is used instead.  See :ref:`date_concise_formatter` for more examples.
+
+plt.rcParams['date.converter'] = 'concise'
+
+fig, ax = plt.subplots(figsize=(5.4, 2), layout='constrained')
+time = np.arange('1980-01-01', '1980-06-25', dtype='datetime64[D]')
+x = np.arange(len(time))
+ax.plot(time, x)
+
+# %%
+#
+# We can set the limits on the axis either by passing the appropriate dates as
+# limits, or by passing a floating-point value in the proper units of days
+# since the epoch.  If we need it, we can get this value from
+# `~.dates.date2num`.
+
+fig, axs = plt.subplots(2, 1, figsize=(5.4, 3), layout='constrained')
+for ax in axs.flat:
+    time = np.arange('1980-01-01', '1980-06-25', dtype='datetime64[D]')
+    x = np.arange(len(time))
+    ax.plot(time, x)
+
+# set xlim using datetime64:
+axs[0].set_xlim(np.datetime64('1980-02-01'), np.datetime64('1980-04-01'))
+
+# set xlim using floats:
+# Note can get from mdates.date2num(np.datetime64('1980-02-01'))
+axs[1].set_xlim(3683, 3683+60)
+
+# %%
+#
+# String conversion: categorical plots
+# ====================================
+#
+# Sometimes we want to label categories on an axis rather than numbers.
+# Matplotlib allows this using a "categorical" converter (see
+# `~.matplotlib.category`).
+
+data = {'apple': 10, 'orange': 15, 'lemon': 5, 'lime': 20}
+names = list(data.keys())
+values = list(data.values())
+
+fig, axs = plt.subplots(1, 3, figsize=(7, 3), sharey=True, layout='constrained')
+axs[0].bar(names, values)
+axs[1].scatter(names, values)
+axs[2].plot(names, values)
+fig.suptitle('Categorical Plotting')
+
+# %%
+#
+# Note that the "categories" are plotted in the order that they are first
+# specified and that subsequent plotting in a different order will not affect
+# the original order.  Further, new additions will be added on the end (see
+# "pear" below):
+
+fig, ax = plt.subplots(figsize=(5, 3), layout='constrained')
+ax.bar(names, values)
+
+# plot in a different order:
+ax.scatter(['lemon', 'apple'], [7, 12])
+
+# add a new category, "pear", and put the other categories in a different order:
+ax.plot(['pear', 'orange', 'apple', 'lemon'], [13, 10, 7, 12], color='C1')
+
+
+# %%
+#
+# Note that when using ``plot`` like in the above, the order of the plotting is
+# mapped onto the original order of the data, so the new line goes in the order
+# specified.
+#
+# The category converter maps from categories to integers, starting at zero. So
+# data can also be manually added to the axis using a float.  Note that if a
+# float is passed in that does not have a "category" associated with it, the
+# data point can still be plotted, but a tick will not be created.  In the
+# following, we plot data at 4.0 and 2.5, but no tick is added there because
+# those are not categories.
+
+fig, ax = plt.subplots(figsize=(5, 3), layout='constrained')
+ax.bar(names, values)
+
+# 0 gets labeled as "apple"
+ax.plot(0, 0, 'd', color='C1')
+ax.text(0, 0, '  Float x=0', rotation=45, color='C1')
```

Comment:
```suggestion
ax.text(0, 0, '  Float x=0', rotation=45, color='C1', bbox={'color': 'gray', 'alpha': .2, 'boxstyle': 'round'})
```

I would stick a bounding box on all of these.  The thin orange text is really hard to read against the blue/white background.